data/yaml/mspass.yaml

```diff
@@ -429,13 +433,10 @@ Metadata:
         readonly: false
       site_id:
         collection: wf_Seismogram
-        readonly: false
-      channel_id:
-        collection: wf_Seismogram
-        readonly: false
```

Comment:
The `channel_id` is a list defined in `wf_Seismogram`. It should not be removed here.